provider/provider.go

```diff
@@ -74,6 +75,10 @@ func (p *dockerNativeProvider) DiffConfig(ctx context.Context, req *rpc.DiffRequ
 
 // Configure configures the resource provider with "globals" that control its behavior.
 func (p *dockerNativeProvider) Configure(_ context.Context, req *rpc.ConfigureRequest) (*rpc.ConfigureResponse, error) {
+	// TODO: ensure we are also setting env vars here
```

Comment:
do we need to fix this TODO for this PR?

Reply:
Nope, I just forgot to remove this, it was a personal reminder